NativeDemo/TokamakDemo.xcodeproj/project.pbxproj

```diff
@@ -490,7 +490,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 288H3WAR3W;
```

Comment:
Is this an issue/should I revert?

Reply:
Not an issue for me, as long as it builds on CI 🙂

Reply:
Seems like it builds fine 👍